css-paint-api/Overview.bs

```diff
@@ -451,8 +451,12 @@ Note: The user agent can optionally defer drawing images which are <a>paint-inva
 </div>
 
 The <a>draw a paint image</a> function is invoked by the user agent during the <a>object size
-negotiation</a> algorithm which is responsible for rendering an <<image>>, with |concreteObjectSize|
-set to the <a>concrete object size</a> of the <a>box</a>.
+negotiation</a> algorithm which is responsible for rendering an <<image>>, with |snappedConcreteObjectSize|
+defined as follows. Let |concreateObjectSize| be the <a>concrete object size</a> of the <a>box</a>, and
+|deviceObjectSize| be the |concreateObjectSize| multiplied by the {{PaintWorkletGlobalScope/devicePixelRatio}}.
+If the |deviceObjectSize| is non-integral, then adjust it to align with pixel boundary. Finally, set
```

Comment:
It's not a requirement to snap, but the browser is allowed to. How about something like this?

The |snappedConcreteObjectSize| is usually the same as the |concreteObjectSize|; however, the user agent may adjust the size such that it paints to pixel boundaries. If it does, the user agent should adjust the |snappedConcreteObjectSize| by the proportional change from its original size so that the paint function can adjust the drawing accordingly.

Reply:
Done.